src/test/java/org/graylog/integrations/notifications/types/SlackClientTest.java

```diff
@@ -49,14 +40,38 @@ public void tearDown() throws IOException {
     }
 
 
-
     @Test
     public void send_is_successful() throws Exception {
```

Comment:
Better test case names will allow you to know what is broken with just a look at the JUnit results.
```suggestion
    public void send_sendsHttpRequestAsExpected_whenInputIsGood() throws Exception {
```